examples/index.ts

```diff
@@ -139,6 +156,28 @@ async function doRequests(provider: BaseProvider) {
     console.info(" ")
 }
 
+async function doSolanaRequests(provider: BaseProvider) {
```

Comment:
on further review, `SolanaRequests` doesn't make sense. You mean `SwapRequests` or `AmmRequests` right?